components/contract-functions/contract-function.tsx

```diff
@@ -33,13 +34,17 @@ export const ContractFunction: React.FC<ContractFunctionProps> = ({
     return null;
   }
 
+  const isFunction = "stateMutability" in fn;
```

Comment:
I think I would prefer if we directly pass through if were in "function" or "event" mode